stable/redis-ha/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+name: redis-ha
+home: http://redis.io/
+engine: gotpl
+keywords:
+- redis
+- keyvalue
+- database
+version: 0.2.0
+description: Highly available Redis cluster with multiple sentinels and standbys.
+maintainers:
+  - email: ikaboubi@gmail.com
+    name: KABOUBI
+details:
+  This package provides a highly available Redis cluster with multiple sentinels and standbys.
+  Note the `redis-master` pod is used for bootstrapping only and can be deleted once
+  the cluster is up and running.
```

Comment:
My apologies. I had tried that but didn't verify that it completed the election.